scrapi/harvesters/erudit.py

```diff
@@ -1,8 +1,7 @@
-# coding=utf-8
```

Comment:
probably not the best way to correct for this, but I just copied the previous version to overwrite my changes and added it to the pull request. Will try to revert next time.

Also changed erudit to done to avoid confusion later.

Sorry!